lms/templates/courseware/progress.html

```diff
@@ -18,6 +18,7 @@
 <%namespace name="progress_graph" file="/courseware/progress_graph.js"/>
 
 <%block name="pagetitle">${_("{course_number} Progress").format(course_number=course.display_number_with_default) | h}</%block>
+<%block name="nav_skip">#content</%block>
```

Comment:
That's an interesting question. This is how the template was before @clrux's change, so it should behave as before unless that was removed as part of his other changes. @clrux, let us know what you think will happen here.